lib/models/device.coffee

```diff
@@ -88,9 +88,32 @@ getDeviceModel = (deps, opts) ->
 		if semver.lt(version, minVersion)
 			throw new Error("Incompatible supervisor version: #{version} - must be >= #{minVersion}")
 
-	addApplicationName = (device) ->
+	###*
+	# @summary Get Dashboard device URL for a device
+	# @function getDashboardUrl
+	#
+	# @param {Object} options - options
+	# @param {String} options.appId - Application id
+	# @param {String} options.deviceId - Device id
+	#
+	# @returns {String} - Dashboard URL
+	# @throws Exception if either appId or deviceId are empty
+	#
+	# @example
+	# dashboardUrl = resin.models.device.getDashboardUrl({ appId: '012345', deviceId: '678901' })
```

Comment:
Can you use numbers instead of strings for the docs and all the examples here?

Doesn't matter if we happen to support both anyway, but we use numbers only for ids everywhere else in the SDK (we interpret strings as being UUIDs/app names, which we send to the API to get the corresponding ids when required). It would be good to be consistent with that, and having the documented API specifically require numbers leaves us open to potentially add UUID/app name support later or even https://github.com/resin-io/resin-sdk/issues/280 for total consistency, without it technically being a breaking change.

Maybe we should even mention that actually just to warn people: `@description Note that while passing strings for ids may work here, this will potentially break in future releases.`